packages/inbound-mail-parser/src/parser.spec.js

```diff
@@ -31,7 +31,7 @@ describe('test_parse', () => {
       const parse = new Parse({}, {});
 
       function callback(email) {
-        expect(email).to.be.null;
+        expect(email).to.be.null();
```

Comment:
This is not right, `null` isn't a function so should not be called

Reply:
I believe the function form should work as dirty-chai is set up by Mocha: https://github.com/sendgrid/sendgrid-nodejs/blob/master/test/mocha.js

However, I've reverted the change and added an `eslint-disable-next-line` so the original implementation is retained.

Reply:
Yeah you are correct, I missed that!

Reply:
I've re-added the original change and removed the `eslint-disable-next-line` for consistency as the dirty-chai function form is used in other tests in the project.